### PR TITLE
Fix build, update imports

### PR DESCRIPTION
**Pull request by stephenkirk-toast**
This PR:
- Updates the mac signing cert with a temp apple account cert. (There was a circle-ci breach that made this necessary). 
- Updates all gradle imports, compose and gradle wrapper to the latest version. Replaced a deprecated compose function brought in from the compose upgrade as well
- Update to java 17 for packageDeb to work
- Fix previously missed broken tests

### CODE REVIEW

File: src/main/kotlin/com/toasttab/pulseman/view/TabHeaderUI.kt [hunk 1]
```diff
@@ -18,7 +18,7 @@ package com.toasttab.pulseman.view
 import androidx.compose.foundation.HorizontalScrollbar
 import androidx.compose.foundation.background
 import androidx.compose.foundation.clickable
-import androidx.compose.foundation.gestures.forEachGesture
+import androidx.compose.foundation.gestures.awaitEachGesture
```
Comment (stephenkirk-toast) [inline src/main/kotlin/com/toasttab/pulseman/view/TabHeaderUI.kt:21]: This was deprecated in the latest version of compose

File: .circleci/config.yml [hunk 2]
```diff
@@ -49,17 +49,26 @@ jobs:
...
-          name: "Run tests"
+          name: "Install fakeroot"
+          command: |
+            sudo apt-get update
+            sudo apt-get install fakeroot
+      - run:
+          name: "Run test"
           command: ./gradlew test
+      - store_test_results:
+          path: ~/repo/build/test-results/test/
```
Comment (stephenkirk-toast) [inline .circleci/config.yml:60]: Wrong folder was being used before, moving up as well to make sure they are copied if a follow up step fails

File: .circleci/config.yml [hunk 3]
```diff
@@ -49,17 +49,26 @@ jobs:
...
       - run:
           name: "Create Package deb"
-          command: ./gradlew packageDeb
-      - write_cache_linux
-      - store_test_results:
-          path: ~/repo/build/test-results/iTest/
+          command: ./gradlew packageDeb --info
```
Comment (stephenkirk-toast) [inline .circleci/config.yml:63]: Added info to get more debug info if a build fails

File: .circleci/config.yml [hunk 4]
```diff
@@ -77,13 +86,13 @@ jobs:
             at: artifacts
       - run:
           name: "Decode signing cert"
-          command: base64 -D -o cert.p12 \<<< $SIGNING_CERT
+          command: base64 -D -o cert.p12 \<<< $SIGNING_CERT_TEMP
```
Comment (stephenkirk-toast) [inline .circleci/config.yml:89]: Putting in temp vars to point at a new account, I can't get the company cert to work.

File: .circleci/config.yml [hunk 5]
```diff
@@ -96,17 +105,25 @@ jobs:
           name: "Check current java version"
           command: java -version
       - run:
-          name: "Install java 15"
+          name: "Install java 17"
           command: |
-            brew tap adoptopenjdk/openjdk
-            brew install adoptopenjdk15
```
Comment (stephenkirk-toast) [inline .circleci/config.yml:102]: This project is now dead. Java 17+ not supported
New import requires a symlink

**Review by jamiecondon-toast — APPROVED** [review]

File: .circleci/config.yml (hunk 5 above)
Comment (jamiecondon-toast) [reply .circleci/config.yml:102]: Nice workaround

**Pull request merged**